lib/plugin.js

```diff
@@ -4,18 +4,26 @@ import middleware from '@@/.nuxt/middleware'
 const moduleName = '<%= options.moduleName %>'
 
 const initStore = async context => {
-  if (process.client) return
```

Comment:
We need to token saved on the client side for refreshing - there is no sensitive information displayed.

Reply:
As long as this is run on server side it should be there on client side—I didn't think this needs to run on both.

Reply:
nuxt will only carry state to the client (obviously passing functions would be magic) so we nee to register the module on the client to use mutations or actions etc..